### DISCUSSION TITLE
Which PVCs are so important that they should be replicated?

### DISCUSSION THREAD

**Discussion by mrclrchtr** [grafana/mimir#8302] [Help and support] [answered]
Mimir requires a relatively large amount of hard disk space. Our standard StorageClass replicates the data to provide fail-safety.

To save disk space, we would like to avoid having to replicate the PVCs that are stateless.

For example, it is not necessary for the compactor, is it?

**Comment by charleskorn** [comment, marked as answer]
Definitely not necessary for compactors and store-gateways: if these PVCs are lost, compactors and store-gateways will just take longer to start up and begin doing useful work.

For ingesters, provided you have a replication factor greater than 1, it'd also be OK to lose an ingester PVC now and then, as the data should be replicated internally to other ingesters. However, you should ensure that ingester PVCs are preserved when ingesters restart or are rescheduled to other nodes, as otherwise doing something like an upgrade could result in data loss.

**Reply by mrclrchtr** [reply]
Thank you very much!
